### PR TITLE
User Profile Backend

### PR DESCRIPTION
**Pull request by Meet-Tatva**
-User Profile Backend

### CODE REVIEW

File: CI-API.Application/ServiceInterface/IUserProfileService.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CI_API.Application.ServiceInterface
+{
+    public interface IUserProfileService
+    {
+        public Task<JsonResult> GetCountriesList();
```
Comment (wilmanpatel) [inline CI-API.Application/ServiceInterface/IUserProfileService.cs]: This should not be a part of UserProfileService and should be moved to other new Service

Comment (Meet-Tatva) [reply CI-API.Application/ServiceInterface/IUserProfileService.cs]: Changes Done

**Review by wilmanpatel — APPROVED** [review]

**Review by wilmanpatel — APPROVED** [review]

File: CI-API.Data/Repository/CommonRepository.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    public class CommonRepository:ICommonRepository
+    {
+        private readonly CiPlatformDbContext cIDbContext;
+        public CommonRepository(CiPlatformDbContext _cIDbContext)
+        {
+            cIDbContext = _cIDbContext;
+        }
+
+        public async Task<JsonResult> GetCountriesList()
+        {
```
Comment (ShubhamPadiyaTatvasoft) [inline CI-API.Data/Repository/CommonRepository.cs:24]: use try-catch block

File: CI-API/Controllers/CommonController.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    [ApiController]
+    public class CommonController : ControllerBase
+    {
+        ICommonService commonService;
+        public CommonController(ICommonService _commonService)
+        {
+            commonService = _commonService;
+        }
+
+        [HttpGet("GetCountryList")]
```
Comment (ShubhamPadiyaTatvasoft) [inline CI-API/Controllers/CommonController.cs:17]: use try-catch block

File: CI-API.Data/Repository/CommonRepository.cs (hunk 2 above)
Comment (Meet-Tatva) [reply CI-API.Data/Repository/CommonRepository.cs:24]: Try and Catch Block Added

**Review by ShubhamPadiyaTatvasoft — APPROVED** [review]

**Review by UrvishaNayee — APPROVED** [review]

**Pull request merged**